elf.items =items

```diff
@@ -0,0 +1,33 @@
+[33m2e6f221[m[33m ([m[1;36mHEAD -> [m[1;32mExercise3[m[33m, [m[1;31morigin/master[m[33m, [m[1;31morigin/HEAD[m[33m, [m[1;31morigin/Excercise1[m[33m, [m[1;32mmaster[m[33m, [m[1;32mExcercise1[m[33m)[m HEAD@{0}: checkout: moving from Exercise2 to Exercise3
```

Comment:
I'm not sure what this file is but we shouldn't need it - let's delete it (and the one called `s ViewModel`)